package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "opstooling-integrations",
-  "version": "1.0.0",
-  "description": "Reusable third party integrations library",
+  "name": "@eng-automation/integrations",
+  "version": "0.0.0-updated-via-gh-releases",
+  "description": "Parity EngAutomation reusable third party integrations",
   "main": "dist/index.js",
   "author": "Parity <admin@parity.io> (https://parity.io)",
   "license": "Apache-2.0",
   "dependencies": {
+    "@eng-automation/js": "^0.0.21",
```

Comment:
since it's not stable, probably should go hardcoded without `^`, as any version has potentially breaking changes, right?

Reply:
With `0.0.x`, it'll basically behave like fixed version anyway, so no sudden updates, unless it's `^0.0.21-patch0`, or something
![Screen Shot 2023-04-11 at 12 30 42](https://user-images.githubusercontent.com/588262/231134333-f65a5fda-b74e-4b0c-bd85-087433790313.png)

Reply:
ah, thanks, good to know